matcher/service/celery_tasks.py

```diff
@@ -38,7 +38,7 @@ def on_task_failure(self, exc, task_id, args, kwargs, einfo):
     max_retries=15,
     retry_jitter=True,
 )
-def set_error_status(self, interface: ConfigNoteInterface, logger, exc):
+def set_error_status(self, interface: BaseConfigNoteInterface, logger, exc):
```

Comment:
why do you need to define a the type here?

Reply:
I'm not sure, I think it's optional styling - it should work just fine if it was removed as well

Reply:
we usually don't define the type and it is easier to have different objects representing the same parameter


if you remove the type then the different interfaces don't need to inherit from the same class, they can be just different classes but following the same interface.